src/components/embed/ui/molecules/copy-to-clipboard/CopyToClipboard.tsx

```diff
@@ -0,0 +1,92 @@
+import { useEffect, useState } from "react";
+import styled from "styled-components";
+import browser from "webextension-polyfill";
+import copy from "copy-to-clipboard";
+import { Check, Copy02 } from "@untitled-ui/icons-react";
+import { toast } from "react-toastify";
+import { Text } from "~components/embed/ui";
+
+interface CopyToClipboardProps {
+  text: string;
+  copySuccess?: string;
+  iconSize?: number;
+  label?: string;
+  labelStyle?: React.CSSProperties;
+  labelAs?: typeof Label;
+  showToast?: boolean;
+  onCopy?: (isCopied: boolean) => void;
+}
+
+const CopyButton = styled.button`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  cursor: pointer;
+  border: none;
+  background: none;
+  gap: 8px;
+`;
+
+const Label = styled(Text).attrs({
+  noMargin: true,
+})`
+  overflow: hidden;
+`;
+
+export function CopyToClipboard({
```

Comment:
There's currently a copy button in the account selector, another one in the backup seedphrase view and another one in the backup QR view. Now that we have this component, they should all use it. I think the root element should be the `InputButton` component though.